MVC-App/Controllers/HomeController.cs

```diff
@@ -26,5 +29,21 @@ public ActionResult Contact()
 
             return View();
         }
+
+        public ActionResult RelationView()
+        {
+            //using (testEntities db = new testEntities())
+            //{
+            //    //tblRelation relation = db.tblRelation.Where(p => Int32.Parse(p.Id.ToString()) == id).FirstOrDefault();
+            //    return View(db.tblRelation);
+            //}
+            return View(db.tblRelation);
+        }
+
```

Comment:
If this code in usless just remove it, not comment it.

Reply:
this code may be useful in next features, so I kept it